packages/sanity/src/core/store/_legacy/document/document-pair/utils/actionsApiClient.ts

```diff
@@ -3,7 +3,7 @@ import {type SanityClient} from '@sanity/client'
 import {RELEASES_STUDIO_CLIENT_OPTIONS} from '../../../../../releases/util/releasesClient'
 import {type IdPair} from '../../types'
 
-const ACTIONS_API_MINIMUM_VERSION = '2024-05-23'
+const ACTIONS_API_MINIMUM_VERSION = 'X'
```

Comment:
Let's remember to change this once the API is deployed.

Reply:
This can now be changed to `2025-02-19` @skogsmaskin

Reply:
I wonder, too, if this really needs to be bumped? @sgulseth

Reply:
I checked up on this now, and we do need it to be able to write GDRs. Otherwise the mutation will error.

Reply:
Ok, but we can't merge this using vX unflagged, since that will affect everyone

Reply:
Changing to `v2025-02-19` is fine

Reply:
And would appreciate a rename + a comment about why it requires that as a minimum version ;)